typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -157,16 +184,63 @@ export class HyperlaneIsmFactory extends HyperlaneApp<ProxyFactoryFactories> {
     return IMultisigIsm__factory.connect(address, signer);
   }
 
-  private async deployRoutingIsm(chain: ChainName, config: RoutingIsmConfig) {
+  private async deployFallbackRoutingIsm(params: {
```

Comment:
I would probably attempt to reuse the `deployRoutingIsm` function here
afaict its identical except for the factory